sp/src/game/server/hl2/npc_fastzombie.cpp

```diff
@@ -71,6 +76,10 @@ ConVar sk_zombie_fast_dmg_one_slash( "sk_zombie_fast_dmg_claw","3");
 ConVar sk_zombie_fast_dmg_both_slash( "sk_zombie_fast_dmg_leap","5");
 #endif
 
+#ifdef EZ2
+ConVar	sk_zombie_fast_kick_multiplier( "sk_zombie_fast_kick_multiplier", "0" );
```

Comment:
Am I right to assume this is also changed by a skill/autoexec file?

Reply:
Yep!